packages/config/generateCitations.js

```diff
@@ -0,0 +1,89 @@
+import "@citation-js/plugin-bibtex";
+import "@citation-js/plugin-software-formats";
+import "@citation-js/plugin-csl";
```

Comment:
did we end up needing all three of these plugins?

Reply:
yes